android/src/main/java/com/robinpowered/react/ScreenBrightness/ScreenBrightnessPackage.java

```diff
@@ -14,18 +14,14 @@
 import java.util.List;
 
 public class ScreenBrightnessPackage implements ReactPackage {
-  private final int mWriteSettingsRequestCode;
-
-  public ScreenBrightnessPackage(final int writeSettingsRequestCode) {
```

Comment:
@mars-lan are we able to make the `androidx` compatibility change without changing the package's constructor?

Reply:
Actually the change has nothing to do with `androidx` but more to do with RN 0.60's [autolinking](https://github.com/react-native-community/cli/blob/master/docs/autolinking.md), which requires a no-arg constructor.

Reply:
Hrmm, perhaps we can support two constructors then, where the no-arg constructor sets the default value, but can still be specified if needed.

The `requestCode` is [used to identify the intent](https://stackoverflow.com/a/14148838) that requests permissions. The problem is that the `requestCode` should be unique across the application, and if the package defines it internally, it can conflict with another `requestCode` the user defined for something else.

Reply:
Makes sense. Restored the original constructor.